test_conformance/api/test_kernels.cpp

```diff
@@ -87,6 +87,24 @@ const char *sample_two_kernel_program[] = {
 "\n"
 "}\n" };
 
+const char *sample_read_only_image_test_kernel = {
+    "__kernel void read_only_image_test(__write_only image2d_t img, __global "
+    "uint4 *dst)\n"
```

Comment:
`dst` is never written to, so this should probably be `src`?

```suggestion
    "uint4 *src)\n"
```

Reply:
Done